packages/components/src/styling/colors.ts

```diff
@@ -98,4 +98,23 @@ export const colors = {
     infographic_substitute_pink_rose: "#E24973",
     infographic_substitute_pink_salmon: "#FF92A8",
     infographic_substitute_purple_berry: "#8C1159",
+
+    //most of the toast colors are custom, not from EDS foundation
+    toast_error_background_light: "#FFD2D1",
```

Comment:
What is the reason for these custom colors?

Reply:
The design use these custom colors: https://www.figma.com/design/GmYi1qTwOkAM4ZyK5Bsj31/Component-library?m=dev&node-id=542%3A39794